src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -1426,6 +1450,23 @@ private static void EnableCloseButton(IntPtr hwnd)
                            MF_BYCOMMAND | MF_ENABLED);
         }
 
+        private RECT ClientRectToWindowRect(RECT clientRect, WindowStyles? styleOverride = null, WindowStyles? extendedStyleOverride = null)
```

Comment:
There are several other uses of `AdjustWindowRectEx` which could be redirected to this method. These other locations currently don't account for per-window DPI, so using this method may fix some bugs. But I decided not to make unnecessary changes in this PR.